sdk/containerservice/arm-containerservice/package.json

```diff
@@ -3,83 +3,81 @@
   "sdk-type": "mgmt",
   "author": "Microsoft Corporation",
   "description": "A generated SDK for ContainerServiceClient.",
-  "version": "22.0.0-beta.1",
+  "version": "21.5.0",
   "engines": {
     "node": ">=18.0.0"
   },
   "dependencies": {
-    "@azure/abort-controller": "^2.1.2",
-    "@azure/core-auth": "^1.9.0",
-    "@azure/core-client": "^1.9.3",
     "@azure/core-lro": "^2.5.4",
+    "@azure/abort-controller": "^2.1.2",
     "@azure/core-paging": "^1.6.2",
-    "@azure/core-rest-pipeline": "^1.19.1",
+    "@azure/core-client": "^1.9.2",
+    "@azure/core-auth": "^1.9.0",
+    "@azure/core-rest-pipeline": "^1.19.0",
     "tslib": "^2.8.1"
   },
   "keywords": [
     "node",
     "azure",
     "typescript",
     "browser",
-    "isomorphic"
+    "isomorphic",
+    "cloud"
   ],
   "license": "MIT",
   "main": "./dist/commonjs/index.js",
   "module": "./dist/esm/index.js",
   "types": "./dist/commonjs/index.d.ts",
   "devDependencies": {
-    "@azure-tools/test-credential": "^2.0.0",
-    "@azure-tools/test-recorder": "^4.1.0",
-    "@azure-tools/test-utils-vitest": "^1.0.0",
+    "typescript": "~5.7.2",
+    "dotenv": "^16.0.0",
     "@azure/dev-tool": "^1.0.0",
-    "@azure/identity": "^4.8.0",
+    "@azure/identity": "^4.6.0",
     "@azure/logger": "^1.1.4",
+    "@azure-tools/test-recorder": "^4.1.0",
+    "@azure-tools/test-credential": "^2.0.0",
+    "@azure-tools/test-utils-vitest": "^1.0.0",
     "@types/node": "^18.0.0",
     "@vitest/browser": "^3.0.9",
     "@vitest/coverage-istanbul": "^3.0.9",
-    "dotenv": "^16.0.0",
-    "playwright": "^1.51.0",
-    "typescript": "~5.6.2",
```

Comment:
why is this missing?

Reply:
they are moved to https://github.com/Azure/azure-sdk-for-js/pull/33497/files#diff-6e26c6173caa8be0b5fe471bf9a6dc0c0a9db95e9b3b6707fcc4aa806fc10367R32-R33